module/data/shared/roll-config-field.mjs

```diff
@@ -3,8 +3,8 @@ import AdvantageModeField from "../fields/advantage-mode-field.mjs";
 const { StringField, NumberField, SchemaField } = foundry.data.fields;
 
 /**
- * @typedef {object} RollConfigData
- * @property {string} ability   Default ability associated with this roll.
+ * @typedef {object} RollConfigFieldData
```

Comment:
Make sure you update other usages of this if you are renaming it, it is referenced in `actor/templates/creature.mjs`.

Reply:
Had misread those same ones, should not have been renamed.